src/main/java/duke/util/Parser.java

```diff
@@ -0,0 +1,154 @@
+package duke.util;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+import duke.command.AddTaskCommand;
+import duke.command.Command;
+import duke.command.CommandType;
+import duke.command.DeleteTaskCommand;
+import duke.command.EmptyCommand;
+import duke.command.ExitCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.UpdateStatusCommand;
+import duke.task.DeadlineTask;
+import duke.task.EventTask;
+import duke.task.Task;
+import duke.task.TaskSymbolType;
+import duke.task.TodoTask;
+
+/**
+ * Parser to make sense of user input.
+ */
+public class Parser {
+
+    public static final DateTimeFormatter DATE_TIME_INPUT_FORMAT = DateTimeFormatter.ofPattern("d-M-yy HHmm");
+    public static final DateTimeFormatter DATE_TIME_OUTPUT_FORMAT = DateTimeFormatter.ofPattern("hh:mm a, MMM d, yyyy");
```

Comment:
You might want to add header comments to explain what these fields are used for. I think this applies to the fields in the other classes as well, especially the public ones, even though you already did a great job in the naming of each field to be quite self-evident